static/app/utils/replays/hooks/useCrumbHandlers.tsx

```diff
@@ -16,7 +17,7 @@ function useCrumbHandlers(startTimestampMs: number = 0) {
   const {setActiveTab} = useActiveReplayTab();
 
   const handleMouseEnter = useCallback(
-    (item: Crumb) => {
+    (item: Crumb | NetworkSpan) => {
       if (startTimestampMs) {
         setCurrentHoverTime(relativeTimeInMs(item.timestamp ?? '', startTimestampMs));
```

Comment:
double-checked and the timestamps are in the same format, so no conversion is needed here.